src/util/displayInfo.js

```diff
@@ -5,20 +5,39 @@ L.Control.Info = L.Control.extend({
 
   initialize: function(options) {
     L.Util.setOptions(this, options);
-    this._infoDisplay = L.DomUtil.create('div');
-    this._infoDisplay.classList.add('leaflet-control-info', 'leaflet-bar', 'leaflet-control');
-    this._infoDisplay.style.backgroundColor = '#fff';
-    if(this.options.classname) {
-      this._infoDisplay.classList.add(this.options.classname);
+    this._infoModal = L.DomUtil.create('div');
+    this._infoModal.classList.add('modal', 'fade', 'leaflet-bar', 'leaflet-control-info');
+    $(this._infoModal).attr('role', 'dialog');
```

Comment:
Hi, this is looking really nice. I wanted to suggest you could construct the HTML as strings and it might end up a bit more readable... what do you think? For reference, see this section of another project where a block of HTML/CSS is constructed in a string:

https://github.com/publiclab/inline-markdown-editor/blob/c41cf426bc010b382346ffd31b8d2924570162b7/src/buildSectionForm.js#L2-L14

What's more, it used standard Bootstrap classes rather than unique css styles. This could be helpful for you too because it would add standard margins, padding, etc and it might make your code overall shorter. If you use the standard Bootstrap html/css as in https://getbootstrap.com/docs/4.0/components/modal/, you may not need as many customizations of the style attribute, you know? And it may accommodate more varied content since those styles have been so thoroughly tested. 

What do you think? 

Thanks again for your help!!! 🎉

Reply:
Hi, I think it is a nice idea to construct the HTML as string and as for the unique css styles, i added those because the styles present in the standard bootstrap were being overwritten due to  another class.

Reply:
@jywarren i made the changes you requested but the test are failing and i m not able to find the cause, would you help me out.